examples/flutter_dynamic_form/lib/new_car/bloc/new_car_bloc.dart

```diff
@@ -15,14 +15,14 @@ class NewCarBloc extends Bloc<NewCarEvent, NewCarState> {
 
   final NewCarRepository _newCarRepository;
 
-  void _onEvent(NewCarEvent event, Emitter<NewCarState> emit) {
+  Future<void> _onEvent(NewCarEvent event, Emitter<NewCarState> emit) async {
```

Comment:
```suggestion
  Future<void> _onEvent(NewCarEvent event, Emitter<NewCarState> emit) {
```

Reply:
It won't work without `async` because it's possible that none of the if blocks will be reached and then the function will return void.